embeddings/transformation/hf_transformation/downsample_transformation.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional
+
+import datasets
+
+from embeddings.transformation.transformation import Transformation
+
+
+class DownsampleHuggingFaceTransformation(
+    Transformation[datasets.DatasetDict, datasets.DatasetDict]
+):
+    def __init__(
+        self,
+        downsample_train: Optional[float] = None,
+        downsample_dev: Optional[float] = None,
+        downsample_test: Optional[float] = None,
+        seed: int = 441,
```

Comment:
setting default seed to 441 in many places shouldn't be unified?

Reply:
I like this remark, we can have some kind of LIBRARY_SEED but I would move it to another issue and add to refactor backlog